src/pages/scheduler/components/SchedulerSections.tsx

```diff
@@ -164,16 +172,30 @@ export const Option = forwardRef<OptionsProps, 'div'>(
           </HStack>
           <VStack flexGrow={1} py="1.5">
             {meetingTimes.map((m, key) => (
-              <HStack key={key} w="100%" px="5">
-                <Box w="33%" minW="27%">
+              <HStack key={key} w="100%" px="1">
+                {seats && (
+                  <Box w="20%" minW="10%">
+                    <Flex flexWrap={'wrap'} justify={'center'}>
+                      <Badge as="b" colorScheme={sectionFull ? 'red' : 'green'}>
+                        {seats.enrollment}/{seats.maxEnrollment}
+                      </Badge>
+                      {sectionFull && (
+                        <Badge as="b" colorScheme={waitlistFull ? 'red' : 'green'}>
+                          ({seats.waitCount}/{seats.waitCapacity})
+                        </Badge>
+                      )}
+                    </Flex>
+                  </Box>
+                )}
+                <Box w="20%" minW="20%">
                   {m.time.split('-').map((time) => (
                     <Text key={time}>{time}</Text>
                   ))}
                 </Box>
-                <Box w="20%" minW="13%">
+                <Box w="10%" minW="7.5%">
                   {m.days}
                 </Box>
-                <Box w="47%">
+                <Box w="40%">
                   <Location short={`${m.buildingAbbreviation} ${m.roomNumber}`} long={m.where} />
```

Comment:
If you add the `alwaysShort` prop as `true` to this it could help clear up space and make things less squished?

Reply:
done, i also adjusted the spacing of some columns to make things less squishy